src/DssSpell.t.base.sol

```diff
@@ -1054,8 +1054,6 @@ contract DssSpellTestBase is Config, DssTest {
         // Arbitrary amount of TOKEN to test PSM sellGem and buyGem with (in whole units)
         // `amount` is the amount of _TOKEN_ we are selling/buying (NOT measured in Dai)
         uint256 amount = 100_000;
-        // Amount should be more than 10,000 as `tin` and `tout` are basis point measurements
-        require(amount >= 10_000, "checkPsmIlkIntegration/amount-too-low-for-precision-checks");
```

Comment:
Don't remove this, future tests will change this number and the test for final DAI balance checks < 1 WAD (1 DAI) so if the amount is less than 10K then accuracy of `tin` and `tout` testing will not provide enough coverage (they operate in basis points) and may pass when they should fail (though I don't think that was the case before, a 1000 DAI test amount isn't good).

The accompanying comment you are also deleting explains why this needs to stay in the test and will prevent people from making this mistake in the future